### PR TITLE
Internationalization

### PR DESCRIPTION
**Pull request by ina111**
I am a Japanese engineer.
The ChuanhuChatGPT project is interesting.
It was also featured in a Japanese web article. Several Japanese people have requested to use it in Japanese.
So I created a pull request to internationalize just the UI so that it can switch to English and Japanese.

Internationalization inevitably requires changes to the text part, so I made some changes to ChuanhuChatbot.py.
We will need to enclose the text part in i18n() for future development. This will be a major change in development.

What I am doing,
Read the system language of the PC by the locale module of python, and read it when the language is in the form of en_US.json or ja_JP.json in the locale folder.

The text part of the ChuanhuChatbot.py or in the modules folder is enclosed in i18n(). i18n is an instance of the I18nAuto class in webui_locale.py.

When more languages are added in the future, it can be handled by putting the language json in the locale folder.
Also, it is not destructive and relatively safe because if the original Chinese text is changed, it will just be displayed as it was if the respective language json was not there.

### CODE REVIEW

**Comment by GaiZhenbiao** [comment]
Wow, your implementation is so much more elegant than the previous attempts to internationalize Chuanhu Chat! Thanks for your work! 🚀

File: locale/en_US.json [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    "**发送消息** 或 **提交key** 以显示额度": "**Send message** or **Submit key** to display credit",
+    "选择模型": "Select Model",
+    "选择LoRA模型": "Select LoRA Model",
+    "实时传输回答": "Stream output",
+    "单轮对话": "Single-turn dialogue",
+    "使用在线搜索": "Use online search",
+    "选择回复语言（针对搜索&索引功能）": "Select reply language (for search & index)",
+    "上传索引文件": "Upload index file",
+    "双栏pdf": "Two-column pdf",
+    "识别公式": "OCR",
```
Comment (Keldos-Li) [inline locale/en_US.json:19]: should be "formula OCR" maybe?

Comment (ina111) [reply locale/en_US.json:19]: Indeed, you are right. My mistake.
"formula OCR" is correct.

**Comment by Keldos-Li** [comment]
What you have done is so amazing!!!! Love your work!!!

I think it's a little difficult to maintain if we still have these html code in the i18n file🥲. Maybe there is a way to remove html labels...
https://github.com/GaiZhenbiao/ChuanhuChatGPT/blob/53aae97dd6ddb0ffadf81eaacd8894ce54229a30/modules/presets.py#L54-L62

I'll try to separate constant names for these.

----

btw, where can we read the web article (´・ω・`), I'm really curious... (And may I ask what does 川虎 mean in Japanese?)

**Comment by ina111** [comment]
@Keldos-Li It is not a fundamental solution, but text with HTML tags is now combined with +.

**Comment by Keldos-Li** [comment]
> @Keldos-Li It is not a fundamental solution, but text with HTML tags is now combined with +.

Great work! It'll be fine enough~ I pushed a commit to your branch, now these html labels in translation should be totally removed.

(I also changed the Japanese app title to `Chuanhu Chat🚀`. it will be too long for `Chuanhu Chat 🚀 アムールトラ Chat`. In addition, 川虎 (Chuanhu) can actually be understood as a person's name. (Maybe it is also a good choice to keep the kanji character, such as `川虎Chat 🚀`😀 ))

**Comment by ina111** [comment]
Thanks for the correction! @MZhao-ouo @Keldos-Li 

Chuanhu was also machine translated as Amur tiger, so your advice is helpful.

**Comment by Keldos-Li** [comment]
Everything looks fine now. Is there anything else that needs to be added or modified?

**Comment by Keldos-Li** [comment]
And we would also like to say thank you to @sifue! Thank you for your sharing on the Internet and your previous contribution.

**Comment by ina111** [comment]
There are no additions or corrections from me.
I look forward to this pull request being merged and the Hugging face version being updated.
(I would like to contribute in the future by adding a prompt template.)

**Pull request merged**